tests/ImageSharp.Sandbox46/ImageSharp.Sandbox46.csproj

```diff
@@ -18,19 +18,13 @@
     <ProjectReference Include="..\..\src\ImageSharp\ImageSharp.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="BitMiracle.LibJpeg.NET" Version="1.4.280" />
     <PackageReference Include="Magick.NET-Q16-AnyCPU" Version="7.9.0.1" />
     <PackageReference Include="xunit" Version="2.3.1" />
```

Comment:
Are you able to shed any light into what issues are blocking us upgrading xunit?

Reply:
I spent 30 minutes on this before. I'll take another stab in CQ11.

Reply:
Good man! 👍